create_parameter_weights.py

```diff
@@ -41,32 +155,50 @@ def main():
         default=4,
         help="Number of workers in data loader (default: 4)",
     )
+    parser.add_argument(
+        "--distributed",
+        action="store_true",
+        help="Run the script in distributed mode",
+    )
```

Comment:
Maybe change this into a 0/1 int for True/False, just to be consistent with all the other script. Should maybe change all scripts to use `store_true` at some point, but better to do that all at once so it stays consistent between scripts.